<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>981</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabPlatform">
     <attribute name="title">
      <string>Platform</string>
     </attribute>
     <attribute name="toolTip">
      <string>Show information about OS etc..</string>
     </attribute>
     <widget class="QLabel" name="lblOsIcon">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>140</width>
        <height>140</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblOsName">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>OS Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblCpuIcon">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>140</width>
        <height>140</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblCpuArch">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>210</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CPU Arch</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblRamIcon">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>50</y>
        <width>140</width>
        <height>140</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblRam">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>210</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>RAM</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabProcesses">
     <attribute name="title">
      <string>Processes</string>
     </attribute>
     <attribute name="toolTip">
      <string>Show list of all running processes etc.</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>258</width>
        <height>234</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>140</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Process ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Process Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>User</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>CPU Usage</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Memory Usage</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refreshButton">
         <property name="text">
          <string>Refresh</string>
         </property>
         <property name="shortcut">
          <string>F5</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabResources">
     <attribute name="title">
      <string>Resources</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabFileSystems">
     <attribute name="title">
      <string>File Systems</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>941</width>
        <height>541</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="hlFileSystems"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
